Libplanet/Blocks/Block.cs

```diff
@@ -382,8 +382,8 @@ public BlockHeader Header
 
             // Poor man' way to optimize stamp...
             // FIXME: We need to rather reorganize the serialization layout.
-            byte[] emptyNonce = MakeBlock(new Nonce(new byte[0])).SerializeForHash();
-            byte[] oneByteNonce = MakeBlock(new Nonce(new byte[1])).SerializeForHash();
+            byte[] emptyNonce = MakeBlock(new Nonce(new byte[0])).Header.SerializeForHash();
+            byte[] oneByteNonce = MakeBlock(new Nonce(new byte[1])).Header.SerializeForHash();
```

Comment:
It is just a suggestion, how about making the `MakeBlockHeader()` function which creates a new `BlockHeader`?
`Block<T>.SerializedForHash()` was moved to `BlockHeader` so using `Block<T>.BlockHeader` getter seems useless. I believe it can be related to performance issue a little 🤔

Reply:
It can, but IMHO it isn't slightly significant because `MakeBlock()` calls appear only here during the whole mining...  😕

Reply:
I think there will be a performance drop here, too, but I don't think there will be more performance drops than before.

Reply:
I think there will be a performance drop here, too, but I don't think there will be more performance drops than before.

Reply:
I agree.